x/cronos/keeper/permissions.go

```diff
@@ -0,0 +1,32 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/crypto-org-chain/cronos/x/cronos/types"
+)
+
+const (
+	CanChangeTokenMapping uint64                                  = 1 << iota // 1
+	CanTurnBridge                                                             // 2
+	All                   = CanChangeTokenMapping | CanTurnBridge             // 3
```

Comment:
I guess the intention here is to have a bitvec for the permissions? If so, maybe good to document that intention to prevent potential mistakes (as if there's one more perm added, it should be 0b100 == 4, and all would be 0b111 == 7)

Reply:
added some comments